wayshot/src/wayshot.rs

```diff
@@ -123,40 +124,42 @@ fn main() -> Result<()> {
         }
     };
 
+    let mut buffer = Cursor::new(Vec::new());
```

Comment:
Lets only create the buffer if clipboard is specified. Else this leads to an extra allocation which we can skip. Lets just call image.save here and create the buffer in the clipboard conditional block.

Reply:
I considered that but the stdout block also uses the buffer and I didn't want it to do the buffer write twice in the case of --clipboard + stdout. Should I create the buffer there too?

Reply:
Maybe a function can help